console.rst

```diff
@@ -50,14 +50,14 @@ method. Then you can optionally define a help message and the
     {
         $this
             // the name of the command (the part after "bin/console")
-            ->setName('app:create-users')
+            ->setName('app:create-user')
 
             // the short description shown while running "php bin/console list"
-            ->setDescription('Creates new users.')
+            ->setDescription('Creates a new user.')
 
             // the full command description shown when running the command with
             // the "--help" option
-            ->setHelp('This command allows you to create users...')
+            ->setHelp('This command allows you to create a user...')
```

Comment:
Could I persuade you by comparing the two side by side?
```
'This command allows you to create a user...'
'This command allows to create a user...'
```
For me, the first one is nicer to read. :)